app/helpers/application_helper.rb

```diff
@@ -16,4 +16,22 @@ def render_breadcrumb
       end
     end
   end
+
+  def active_menu_page?(**args)
+    unless args.nil?
+      has_pages = false
+      has_path = false
+
+      has_pages = args[:pages].any? { |page| current_page?(page[:slug]) } unless args[:pages].nil?
+
+      has_path = current_page?(args[:path]) unless args[:path].nil?
```

Comment:
It's not a blocker but a matter of convention comment; `hasSomething` convention in javascript becomes `something?` in ruby - `?` in ruby indicates that a method returns a boolean value (like internal blank?, valid? etc).

Reply:
I'm actually refactoring. So all that will change.

Reply:
👍

Reply:
Trying to make sectors and companies controller actions to also highlight the TPI menu

Reply:
Oh... Didn't go over those... Sorry.